jazz.porque.si/pom.xml

```diff
@@ -6,6 +6,18 @@
 	<groupId>com.caviedes</groupId>
 	<artifactId>jazz.porque.si</artifactId>
 	<version>0.0.3-SNAPSHOT</version>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<configuration>
+					<source>8</source>
+					<target>8</target>
```

Comment:
I configured (into IDE) Java 11, so I thinkg we should go at least to that version

Reply:
Updated to 11 :nerd_face: 
I've seen to my local jdk is: openjdk-15.0.2
Would you like to match this version too?